SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -628,12 +628,20 @@ void CTurbSSTSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container, C
 
       conv_numerics->SetPrimitive(V_domain, V_inlet);
 
+      /*--- Non-dimensionalize Inlet_TurbVars if Inlet-Files are used. ---*/
+      su2double Inlet_Vars[MAXNVAR];
+      Inlet_Vars[0] = Inlet_TurbVars[val_marker][iVertex][0];
+      Inlet_Vars[1] = Inlet_TurbVars[val_marker][iVertex][1];
+      if (config->GetInlet_Profile_From_File()) {
+        Inlet_Vars[0] *= 1 / (config->GetVelocity_Ref() * config->GetVelocity_Ref());
```

Comment:
Trust issues?
```suggestion
        Inlet_Vars[0] /= config->GetVelocity_Ref() * config->GetVelocity_Ref();
```

Reply:
Well, I already got confused quite a bit so I thought I'll just think in multiplication of dimensional values to non-dimensionalize .. I'll edit your suggestion to use the pow(,2) here as well